apps/desktop-wallet/src/features/settings/settingsSlice.ts

```diff
@@ -81,6 +82,9 @@ const settingsSlice = createSlice({
       .addCase(fiatCurrencyChanged, (state, action) => {
         state.fiatCurrency = action.payload
       })
+      .addCase(addressOrderPreferenceChanged, (state, action) => {
+        state.addressOrderPreference = action.payload
+      })
```

Comment:
To save the order preference change in local storage, the `addressOrderPreferenceChanged` action needs to be listed in the listener middleware below. See line `101`.